Doc/whatsnew/3.11.rst

```diff
@@ -176,14 +176,33 @@ The :option:`-X` ``no_debug_ranges`` option and the environment variable
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
+Exception Groups and ``except*`` (PEP 654)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:pep:`654` introduces language features that enable a program
+to raise and handle multiple unrelated exceptions simultaneously.
+The builtin types :exc:`ExceptionGroup` and :exc:`BaseExceptionGroup`
+make it possible to group exceptions and raise them together.
+The new ``except*`` syntax generalizes ``except`` to match subgroups
```

Comment:
In line with our policy that PEPs are historic documents, not documentation, maybe this should deep link to the part under https://docs.python.org/3.11/reference/compound_stmts.html#try where `except*` is described?

Reply:
Both ``:keyword:`except* <except_star>` `` and ``:ref:`except* <except_star>` `` currently link to the beginning of the `try` section, where the `except_star` label is defined.  For a deep link, the label should be moved down to the relevant paragraph.

Reply:
It probably doesn't hurt to include a link to the PEP, to give interested readers the background, rationale and context for the change (as is its purpose), but yeah that should be linked.

Actually, the keyword appears to be linked precisely, both judging from [its position in the source](https://github.com/python/cpython/blob/187949ebf2ae36fcf0817a06f4a7637d0a8b7fc5/Doc/reference/compound_stmts.rst?plain=1#L315) on both 3.12 and 3.11, and the fact that the [index entry](https://docs.python.org/3.11/genindex-E.html) correctly [links](https://docs.python.org/3.11/reference/compound_stmts.html#index-12) to it. 

However, the `:ref:` is indeed not; it would be good to move the non-`try` ones (that presumably don't rely on the section name for default link text, since it doesn't even mention except/else/finally) down to the appropriate passages.

Reply:
I think a link to the top of the section is fine.